tests/tests_change_disk_fs.yml

```diff
@@ -18,19 +18,36 @@
 
     - name: Create a disk device with the default file system type
       include_role:
-        name: storage
+        name: linux-system-roles.storage
       vars:
         storage_volumes:
           - name: test1
             type: disk
             mount_point: "{{ mount_location }}"
             disks: "{{ unused_disks }}"
 
+    - name: Load fstab
+      command: cat /etc/fstab
+      changed_when: false
+      check_mode: no
+      register: _test_change_disk_fs_fstab1
+
+    - name: print fstab device
+      debug:
+        msg: device is: {{ ( _test_change_disk_fs_fstab1.stdout_lines|select('search', pat)
```

Comment:
This string must be quoted because jinja2/yaml doesn't like the colon `:` character in `is:`:
```
Exception when running tests: mapping values are not allowed here
  in "<unicode string>", line 37, column 23:
            msg: device is: {{ ( _test_change_disk_fs_fsta ... 
                                   ^ (line: 37)
```